docs/user/openstack/customization.md

```
@@ -56,11 +59,61 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
 * `zones` (optional list of strings): The names of the availability zones you want to install your nodes on. If unset, the installer will use your default compute zone.
 
 > **Note**
-> The bootstrap node follows the `type`, `rootVolume`, `additionalNetworkIDs`, and `additionalSecurityGroupIDs` parameters from the `controlPlane` machine pool.
+> The bootstrap node follows the `type`, `rootVolume`, `additionalNetworkIDs`, and `additionalSecurityGroupIDs` parameters from the `controlPlane` machine pool. The bootstrap flavor (`type`) can be overridden independently using the cluster-scoped [`bootstrapFlavor`](#cluster-scoped-properties) field.
 
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)